ui/next/app/app.tsx

```diff
@@ -139,7 +140,7 @@ ReactDOM.render(
             <Route path=":table_name" component={ TableDetails } />
           </Route>
         </Route>
-        <Route path="help-us/reporting" component={ HelpUs } />
```

Comment:
After thinking about this, you could put this in the `componentWillMount` or `componentDidMount` of the HelpUs component.